windows/src/desktop/kmshell/main/initprog.pas

```diff
@@ -451,14 +453,14 @@ procedure RunKMCOM(FMode: TKMShellMode; KeyboardFileNames: TStrings; FSilent, FF
       Exit;
   end;
 
-  if not FSilent and (FMode in [fmStart, fmSplash, fmMain, fmAbout]) then   // I4773
+  if not FSilent and (FMode in [fmStart, fmSplash, fmMain, fmAbout, fmBoot]) then   // I4773
```

Comment:
Is this right? Previously, we used `-s` as the boot switch, which meant `FSilent = True`. So I don't think `fmBoot` should be checking TIP install status?

Reply:
Yes, because we were overloading the -s to use it for boot which wasn't a perfect match. For boot we wanted -s to supress splash screen or configuration. but checking the DoCheckTIPInstallStatus? 
I will change it back. DoCheckTIPInstallStatus does nothing anyway as it is all commented out. [DoCheckTIPInstallStatus](https://github.com/keymanapp/keyman/blob/27bf5355c50e1e027c80b79f62bf64c1cf7c09c5/windows/src/desktop/kmshell/main/initprog.pas#L360)